webui/react/package.json

```diff
@@ -28,7 +28,7 @@
     "fp-ts": "^2.11.2",
     "fuse.js": "^6.4.6",
     "hermes-parallel-coordinates": "^0.5.18",
-    "history": "^4.10.1",
+    "history": "^5.2.0",
```

Comment:
https://github.com/remix-run/history/releases/tag/v5.0.0
there is breaking changes but In my investigation, theres no affects.